app/scenes/channel_info/channel_info.js

```diff
@@ -91,18 +102,19 @@ class ChannelInfo extends PureComponent {
         toggleFavorite(currentChannel.id);
     }
 
-    handleLeave() {
+    handleDeleteOrLeave(eventType) {
         const {formatMessage} = this.props.intl;
         const channel = this.props.currentChannel;
         const term = channel.type === Constants.OPEN_CHANNEL ?
             formatMessage({id: 'mobile.channel_info.publicChannel', defaultMessage: 'Public Channel'}) :
             formatMessage({id: 'mobile.channel_info.privateChannel', defaultMessage: 'Private Channel'});
+        const titleCaseEventType = `${eventType.charAt(0).toUpperCase()}${eventType.slice(1)}`;
```

Comment:
Could you change this to just set the message through an if statement to keep it much cleaner? Using `formatMessage` on a formatted string is hard to follow here. I'd prefer it to be something like
```
let title;
let message;
if (eventType === 'leave') {
    title = {id: 'mobile.channel_info.alertTitleLeaveChannel', defaultMessage: 'Leave {term}'};
    message = {id: ..., defaultMessage: ...};
} else if (eventType === 'delete') {
    ...
}
```

Either that or completely separate it into `handleDelete` and `handleLeave` since there's some duplicated code, but not a ton?